app/assets/javascripts/table.js

```diff
@@ -1,21 +1,29 @@
 $(document).ready(function() {
-  $(".button-to").submit(function(event) {
+  $("form").submit(function(event) {
+    event.preventDefault();
+
     $form = $(event.currentTarget);
 
+    // $('form#button-to' + id + 'asdfadsf')
+
+    var url = $form[0].action
```

Comment:
Missing semicolon.<br>'$form' is not defined.<br>'url' is defined but never used.